### ISSUE TITLE
Network settings: maximising the window is tokenistic

### ISSUE THREAD

**Issue by ghost** [linuxmint/cinnamon#4886]
Maximising the Network settings window (accessed via the panel applet) does not display a greater number of networks but instead pads out the window with lots of blank space. This is not very functional or intuitive. I'd call it a bug.

Linux Mint 17.2 x64 Cinnamon 2.8.

**Comment by leigh123linux** [comment]
You have filed the bug report against the wrong package, please close it and re-file the issue against cinnamon-control-center instead.

**Comment by JosephMcc** [comment]
I agree this could be better.

**Comment by nick-s-b** [comment]
Thanks for merging my bug. I searched but didn't see this one.

This is an extremely annoying bug if you have a lot of interfaces and/or VPNs. It makes it nearly unusable and I've started using the terminal to set my interfaces because of this.

**Comment by icarter09** [comment]
@nick-s-b are you still experiencing this issue?

**Comment by nick-s-b** [comment]
@icarter09 Yes, the bug still persists. When scaling the window, the container widget does not scale with the window in the **vertical** direction. It scales fine horizontally and it expands with the window.

![2019-12-07-013039_3840x2160_scrot-fs8](https://user-images.githubusercontent.com/7957590/70370172-ca34bd80-1891-11ea-8351-e16c754fb9ed.png)
